src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -35,8 +35,10 @@ private Configuration() {
         input = new FileInputStream(new File("config.yml"));
       } catch (FileNotFoundException ex) {
         try {
+//          LOG.info("CANNOT FIND FILE CONFIG.YML", ex);
           input = new FileInputStream(new File("config_sample.yml"));
         } catch (FileNotFoundException ex2) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)